views/matrixsplit.html

```diff
@@ -233,8 +233,8 @@
             =
             \begin{bmatrix}
```

Comment:
hit up http://www.wolfram.com/raspberry-pi/ and register your board number for some free Mathematica handiness =)